qiskit_experiments/data_processing/processor_library.py

```diff
@@ -23,20 +23,41 @@ def get_processor(
     meas_level: MeasLevel = MeasLevel.CLASSIFIED,
     meas_return: str = "avg",
     normalize: bool = True,
+    init_qubits: bool = True,
+    memory: bool = False,
+    rep_delay: float = None,
+    num_qubits: int = None,
 ) -> DataProcessor:
     """Get a DataProcessor that produces a continuous signal given the options.
 
     Args:
         meas_level: The measurement level of the data to process.
         meas_return: The measurement return (single or avg) of the data to process.
         normalize: Add a data normalization node to the Kerneled data processor.
+        init_qubits: If False, the qubits are not reset to the ground state after a measurement.
+        memory: If True, single-shot measurement bitstrings are returned.
+        rep_delay: The delay between a measurement and the subsequent circuit.
+        num_qubits: The number of qubits.
 
     Returns:
         An instance of DataProcessor capable of dealing with the given options.
 
     Raises:
         DataProcessorError: if the measurement level is not supported.
     """
+
+    # restless data processing.
+    if meas_level == MeasLevel.CLASSIFIED and not init_qubits and memory and rep_delay < 100e-6:
```

Comment:
The hard-coded value for `rep_delay` is a bit arbitrary. Can we add this in the `analysis_options` that we will pass to `get_processor`?